addons/Rakugo/lib/systems/Parser.gd

```diff
@@ -105,9 +105,9 @@ func _ready():
 
 func parse_script(script:String) -> Dictionary:
 	# Find all dialogue first
-	var current_dialogue: = []
+	var init := [] 
```

Comment:
Why create var init ?

```current_dialog := []``` and use it

Reply:
this is strange, just open vscode and there I have it right:
```cs
var init := [] 
var current_dialogue: = init
 ```

But I don't have any changes to commit 😮
fixed in new commit😄